### PR TITLE
make nix-shell work with new vscode

### PR DESCRIPTION
**Pull request by QAston**
Tested: https://github.com/ChorusOne/hopper/pull/153/files

### CODE REVIEW

**Comment by jenpaff** [comment]
What's the _new vscode version_ ? I'm on Version: 1.64.2 . Just to double check before i test the branch

**Comment by QAston** [comment]
1.64 is the new version where they changed the location of the startup script I think

File: Dockerfile [hunk 1]
```diff
@@ -29,10 +29,10 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     && /bin/bash /tmp/library-scripts/docker-debian.sh "true" "/var/run/docker-host.sock" "/var/run/docker.sock" "${USERNAME}" "${USE_MOBY}" ${DOCKER_VERSION} \
     ## install docker-in-docker daemon (not running by default, but it's there as an option)
     && /bin/bash /tmp/library-scripts/docker-in-docker-optional.sh "true" "${USERNAME}" "true" ${DOCKER_VERSION} \
-    # install minikube, kubectl & helm 
+    # install minikube, kubectl & helm
     && /bin/bash /tmp/library-scripts/kubectl-helm-debian.sh ${KUBECTL_VERSION} ${HELM_VERSION} ${MINIKUBE_VERSION} "automatic" "automatic" "automatic" "${USERNAME}" \
     ## install gh cli
-    && bash /tmp/library-scripts/github-debian.sh "2.4.0" \
+    && bash /tmp/library-scripts/github-debian.sh "2.5.1" \
```
Comment (jenpaff) [inline Dockerfile:35]: when I build the container I get the following error 

> #7 154.3 E: Version '2.5.1' for 'gh' was not found

Comment (QAston) [reply Dockerfile:35]: Yeah, they released a new version in the meantime and the script seems to break in that case. I've changed this to just use latest

**Review by jenpaff — APPROVED** [review]
finally got round to testing this, works fine for me !

**Pull request merged**